memory-viz/bin/cli.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+const { draw } = require("memory-viz");
+
+// Check for correct number of arguments
+if (process.argv.length !== 3) {
+    console.error(
+        "Error: wrong number of arguments.\nProper input: npx memory-viz <path-to-file>"
```

Comment:
"Proper input" -> "Usage"

More importantly, `npx` is only necessary when the package is not installed. When the package is already installed, `memory-viz <path-to-file>` should suffice. Please test that out, as it does have implications for the other sections (and testing)